### PR TITLE
Can Select Dice

### LINKED ISSUE #20: User can choose dice.
User can select up to 3 dice they own that can be rolled.

### PR DESCRIPTION
**Pull request by Zaskoda**
closes #20

### CODE REVIEW

File: client/src/components/_die.svg.vue [hunk 1]
```diff
@@ -1,78 +1,102 @@
...
-      type: Number,
-      default: 1
-    },
-    color: {
-      type: Number,
-      default: 1
-    }
+  },
+  setup () {
+    this.tick()
```
Comment (excalq) [inline client/src/components/_die.svg.vue:17]: Let's add a comment about why this `tick`s, for clarity.

File: client/src/components/_die.svg.vue [hunk 2]
```diff
@@ -1,78 +1,102 @@
...
-      <dten-generic-three v-if="value==2" />
-      <dten-generic-three v-if="value==3" />
-      <dten-generic-three v-if="value==4" />
-      <dten-generic-three v-if="value==5" />
-      <dten-generic-three v-if="value==6" />
-      <dten-generic-three v-if="value==7" />
-      <dten-generic-three v-if="value==8" />
-      <dten-generic-three v-if="value==9" />
-      <dten-generic-three v-if="value==10" />
+  <g  v-if="store.ownedDice[diceid] != null">
```
Comment (excalq) [inline client/src/components/_die.svg.vue:71]: Extra space

File: client/src/web3dice.js [hunk 3]
```diff
@@ -72,32 +74,44 @@ export const web3dice = {
...
     } catch (err) {
       console.log("Error: ", err)
       return 'failed'
     }
-    store.diceTraits[diceId] = traits
-    return traits
+  },
+
+  async getOwnedDice() {
+    const owned = [1,2,1,2,1,2]
```
Comment (excalq) [inline client/src/web3dice.js:112]: Is this just a dummy entry?

**Review by excalq — APPROVED** [review]
nit: I recommend documenting some of the mechanics, like the tick() section.
But still stamping for release.

File: client/src/web3dice.js (hunk 3 above)
Comment (Zaskoda) [reply client/src/web3dice.js:112]: Yes, until I can pull a real list of owned items.

**Pull request merged**